go-selinux/label/label_selinux.go

```diff
@@ -25,7 +25,7 @@ var ErrIncompatibleLabel = fmt.Errorf("Bad SELinux option z and Z can not be use
 // the labels.  The labels returned will include a random MCS String, that is
 // guaranteed to be unique.
 func InitLabels(options []string) (string, string, error) {
-	if !selinux.SelinuxEnabled() {
+	if !selinux.GetEnabled() {
```

Comment:
The problem is we have a constant called selinux.Enabled already. Which is something a caller might do.

if selinux.GetEnabled() == selinux.Enabled ...